Makefile

```diff
@@ -2,9 +2,9 @@
 # Makefile to generate specifications
 #
 
-.PHONY: clean all json franca csv binary c tests travis_targets
+.PHONY: clean all travis_targets json franca csv tests binary protobuf ocf c install deploy
 
-all: clean json franca csv binary c tests
+all: clean json franca csv binary tests protobuf
```

Comment:
Is protobuf supposed to work as of today? (I got a build error, but that could be on my system only)
If it is supposed to be working, I suggest to add it to "travis_targets" as well so that it is included in travis tests automatically

Reply:
On the other hand - protobuf/ocf are in contrib directory so maybe not make sense to require that they shall pass

Reply:
I experimented a little bit with "optional build targets" in #260. If we think it is a good idea that we try to run c, ocf and protobuf on every PR just to track their status, but still wants Travis to ignore whether they fail, then that could be a solution. But this is something we can do later, I think this PR is good to go.